### DISCUSSION TITLE
SQL to file append

### DISCUSSION THREAD

**Discussion by DenisGrosjean** [nextgenhealthcare/connect#6057] [Q&A]
Hello All,

I'm working with Mirth 3.5.1 and I try to use a database Oracle Reader which returns several records that must populate a single file by append each record in one file sent by FTP. My problem is that there are one file per records which erase the previous one. However I've flaged "File Exists" => Append.
Thank for your help

Regard/DG

**Comment by pacmano1** [comment]
@DenisGrosjean, you are using a version of Mirth that  is very old.  Please attempt this on the current release of Mirth.

i’m also converting your issue to a discussion for now until you confirm this in a recent release of mirth. There would not be a fix applied to a version as old as the one you  are on.
